activerecord/lib/active_record/connection_adapters/postgresql/quoting.rb

```diff
@@ -205,14 +205,13 @@ def encode_array(array_data)
           end
 
           def encode_range(range)
-            lower_bound = type_cast_range_value(range.begin)
-            upper_bound = if date_or_time_range?(range)
-              # Postgres will convert `[today,]` to `[today,)`, making it exclusive.
-              # We can use the special timestamp value `infinity` to force inclusion.
-              # https://www.postgresql.org/docs/current/rangetypes.html#RANGETYPES-INFINITE
```

Comment:
Unsure if this comment is still necessary since there's less of a "special case" thing going on -- we're treating both ends of the range the same now.